### PR TITLE
Fix repo build - sign and update artifacts for installation

### PR DESCRIPTION
**Pull request by ajm01**
ci build occurs in 3 phases for now:
* phase 1: run build.sh to build image and copy out the install repo artifacts.
* phase 2: using the artifacts, sign the 3 jar files
1. features/io.openliberty.tools.eclipse.feature_0.1.0.jar
2. plugins/io.openliberty.tools.eclipse.lsp4e_0.1.0.jar
3. plugins/io.openliberty.tools.eclipse.ui_0.1.0.jar
overlay the original versions of those jars with the newly signed jars
* phase 3: run build3.sh - will copy the 3 signed jars into the container, run the mvn -pl command to update the artifacts.xml.xz file with the signing info and then copies that xml file out of the container and into the local repository file structure

### CODE REVIEW

File: ci/Dockerfile [hunk 1]
```diff
@@ -21,9 +21,14 @@ RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
 # Clone the repo.
 RUN mkdir eclipse-ltp \
     && cd eclipse-ltp \
-    && git clone https://github.com/OpenLiberty/liberty-tools-eclipse.git
-
+#    && git clone https://github.com/OpenLiberty/liberty-tools-eclipse.git
+    && git clone https://github.com/scottkurz/liberty-tools-eclipse.git
```
Comment (mezarin) [inline ci/Dockerfile:25]: It looks to me that all of the changes on this file are all designed for testing and should not be merged.

**Review by mezarin — COMMENTED** [review]
Hi Andy, the changes look good. I just had one comment. Also, there is a lot of duplication that hopefully will be eliminated when we automate the signing process also

File: ci/Dockerfile [hunk 2]
```diff
@@ -22,8 +22,12 @@ RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
 RUN mkdir eclipse-ltp \
     && cd eclipse-ltp \
     && git clone https://github.com/OpenLiberty/liberty-tools-eclipse.git
-
+ 
 WORKDIR /eclipse-ltp/liberty-tools-eclipse
+RUN git checkout repo-loc
```
Comment (scottkurz) [inline ci/Dockerfile:27]: Did you mean to delete this?

Comment (ajm01) [reply ci/Dockerfile]: yes, that should have been removed, done

File: ci/Dockerfile (hunk 1 above)
Comment (ajm01) [reply ci/Dockerfile]: agreed, removed

**Review by scottkurz — APPROVED** [review]

**Pull request merged**